src/main/java/com/google/devtools/build/lib/runtime/BlazeModule.java

```diff
@@ -282,13 +283,13 @@ public BuildOptions getDefaultBuildOptions(BlazeRuntime runtime) {
    * @param env the command environment
    * @param request the build request
    * @param buildOptions the build's top-level options
-   * @param configuredTargets the build's requested top-level targets as {@link ConfiguredTarget}s
+   * @param analysisResult the build's requested top-level targets as {@link ConfiguredTarget}s
```

Comment:
this description deserves some rewording, given that the parameter's type changed.

Reply:
Thanks, reworded.